README.md

```diff
@@ -189,17 +185,29 @@ plan myapp::patch (
 
 #### Parameters
 
-##### `targets`
+##### `nodes`
 
 A `TargetSpec` object containing all nodes to wait for.
 
-##### `params`
```

Comment:
Maybe keep the params heading but make it a larger heading `#### params` ?

Reply:
That’s covered by #### Parameters.